src/main/webapp/css/main.css

```diff
@@ -35,6 +35,13 @@ nav a {
   text-decoration: none;
 }
 
+nav form {
```

Comment:
Sorry couldn't find where you used this.

Reply:
It's for the form that the search bar is contained in.